types/mui-datatables/index.d.ts

```diff
@@ -210,113 +207,113 @@ export interface MUIDataTableOptions {
 export type MUIDataTableColumnDef = string | MUIDataTableColumn;
 
 export interface MUIDataTableProps {
-    title: string | React.ReactNode;
     columns: MUIDataTableColumnDef[];
     data: Array<object | number[] | string[]>;
     options?: MUIDataTableOptions;
+    title: string | React.ReactNode;
```

Comment:
Why is this re-ordered?

Reply:
Alphabetized properties inside interface